src/monmove.cpp

```diff
@@ -89,6 +119,39 @@ bool monster::can_move_to( const tripoint &p ) const
         }
     }
 
+    if( has_flag( MF_PATH_AVOID_DANGER ) ) {
```

Comment:
What's the difference between this and the animal check above?

Looks like a lot of duplication, with animal pretty much implying path avoid danger.